src/static/css/cfpb-chart-builder.less

```diff
@@ -60,18 +60,38 @@
     .chart-label();
   }
 
+  .highcharts-label__tile-map {
+    .chart-label();
+    color: @black;
+    font-family: 'AvenirNextLTW01-Demi',Arial,sans-serif;
+    font-weight: bold;
+    span {
+      position: relative !important;
+      white-space: normal !important;
+    }
+  }
+
   &[data-chart-type="tile_map"] {
     .highcharts-data-label {
 
       color: @black;
       font-family: "AvenirNextLTW01-Demi";
+      font-size: 14px;
```

Comment:
This could be updated to `font-size: @size-v` (https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-vars.less#L51)